specification/eventhub/resource-manager/readme.md

```diff
@@ -156,12 +156,15 @@ require: $(this-folder)/../../../profiles/readme.md
 input-file:
   - $(this-folder)/Microsoft.EventHub/stable/2015-08-01/EventHub.json
   - $(this-folder)/Microsoft.EventHub/stable/2014-09-01/EventHub.json
-  - $(this-folder)/Microsoft.EventHub/stable/2017-04-01/AuthorizationRules.json
-  - $(this-folder)/Microsoft.EventHub/stable/2017-04-01/CheckNameAvailability.json
-  - $(this-folder)/Microsoft.EventHub/stable/2017-04-01/consumergroups.json
-  - $(this-folder)/Microsoft.EventHub/stable/2017-04-01/disasterRecoveryConfigs.json
-  - $(this-folder)/Microsoft.EventHub/stable/2017-04-01/eventhubs.json
-  - $(this-folder)/Microsoft.EventHub/stable/2017-04-01/sku.json
+  - $(this-folder)/Microsoft.EventHub/stable/2014-09-01/AuthorizationRules.json
```

Comment:
These should all be referencing api version 2017-04-01, right?

Reply:
my bad, corrected the API version